jni_gen/test/jackson_core_test/lib/com/fasterxml/jackson/core.dart

```diff
@@ -0,0 +1,4100 @@
+// Autogenerated by jni_gen. DO NOT EDIT!
```

Comment:
This file is generated from a Java API with a license, so this generated file must have the same license as a derivative work, and be in a third_party folder.